Sources/BraintreeCore/BTHTTP.swift

```diff
@@ -283,6 +287,10 @@ class BTHTTP: NSObject, NSCopying, URLSessionDelegate {
                 headers["Client-Key"] = key
             }
         }
+        
+        if let additionalHeaders {
+            headers = headers.merging(additionalHeaders) { (_, newHeader) in newHeader}
```

Comment:
We should be able to use the following syntax here:
```suggestion
            headers = headers.merging(additionalHeaders) { $1 }
```